data-and-functions-2/src/functions/getProductsForOrder.js

```diff
@@ -0,0 +1,18 @@
+import getProductById from './getProductById'
+
+const getProductsForOrder = (data, id) => {
+  if (data == null || id == null || data.orders == null) {
+    throw new Error('Missing data or ID')
+  }
+  const order = data.orders.find((o) => o.id === id)
+  if (order === undefined) {

```

Comment:
You can totally do the test the way you have it, but you could also just shorten it to:
```
if (!order) {
```